cabal.haskell-ci

```diff
@@ -1,3 +1,6 @@
 no-tests-no-benchmarks: False
 unconstrained:          False
 irc-channels:           irc.freenode.org#haskell-lens
```

Comment:
For future reference, make sure to also include `irc-if-in-origin-repo: True`.

Reply:
Argh. I always forget that :S

Reply:
Fixed in follow-up PR #89